src/Servers/IIS/AspNetCoreModuleV2/OutOfProcessRequestHandler/serverprocess.cpp

```diff
@@ -106,29 +98,37 @@ SERVER_PROCESS::GetRandomPort
     DWORD  dwExcludedPort = 0
 )
 {
-    HRESULT hr = S_OK;
-    BOOL    fPortInUse = FALSE;
-    DWORD   dwActualProcessId = 0;
+    if (pdwPickedPort == nullptr)
```

Comment:
This one isn't obvious to me, why the change?

Reply:
I've reconsidered this and turned it into a DBG_ASSERT now